CMakeLists.txt

```diff
@@ -14,6 +14,11 @@ cmake_minimum_required(VERSION 3.1)
 
 project(base64 LANGUAGES C VERSION 0.4.0)
 
+cmake_policy(SET CMP0082 NEW)
+cmake_policy(SET CMP0060 NEW)
+cmake_policy(SET CMP0065 NEW)
+cmake_policy(SET CMP0127 NEW)
```

Comment:
This commit is lacking a lot of context. Which warnings are these? Why should they need to be "shut up"? May I suggest adding a bit of explanation to the commit body as to why this change is necessary?

Reply:
Nevermind I guess, this change is removed again by `386bfb`, so I will remove this commit during the rebase.